### PR TITLE
OP-126 - Sylius 1.12 support

### PR DESCRIPTION
**Pull request by rafalmal**

### CODE REVIEW

**Comment by senghe** [comment]
Threads that follow the upgrade topic:

https://github.com/BitBagCommerce/SyliusProductBundlePlugin/pull/39
https://github.com/BitBagCommerce/SyliusProductBundlePlugin/issues/38

**Review by senghe — CHANGES_REQUESTED** [review]

**Comment by jakublech** [comment]
Not much left on this PR. Can you tak a look? It is important, awaited upgrade.

File: composer.json [hunk 1]
```diff
@@ -30,13 +30,12 @@
...
-        "symfony/browser-kit": "^4.4 || ^5.2",
-        "symfony/debug-bundle": "^4.4 || ^5.2",
-        "symfony/dotenv": "^4.4 || ^5.2",
-        "symfony/intl": "^4.4 || ^5.2",
-        "symfony/web-profiler-bundle": "^4.4 || ^5.2",
+        "symfony/browser-kit": "^5.4 || ^6.0",
+        "symfony/debug-bundle": "^5.4 || ^6.0",
+        "symfony/dotenv": "^5.4 || ^6.0",
+        "symfony/intl": "^5.4 || ^6.0",
+        "symfony/web-profiler-bundle": "^5.4 || ^6.0",
```
[the remark this answers is not in the document]
Comment (senghe) [reply composer.json:38]: Yup i know @jakublech. Just saw a 4.4 mention in composer.json, so asked about this. But now it's OK.

File: tests/Application/config/api_resources/Product.xml [hunk 2]
```diff
@@ -0,0 +1,131 @@
+<?xml version="1.0" ?>
```
[the remark this answers is not in the document]
Comment (senghe) [reply tests/Application/config/api_resources/Product.xml:1]: @rafalmal if it's needed, please tell me why - I'm wondering. And if it's necessary, please update the README installation docs to include that files.

[the remark this answers is not in the document]
Comment (rafalmal) [reply tests/Application/config/api_resources/Product.xml:1]: This API resources was located in src/Resources/api_resources earlier, but in sylius 1.12 they stopped beeing imported so i have moved them up in directory tree

**Review by senghe — APPROVED** [review]

**Pull request merged**